### ISSUE TITLE
Run algorithm for different patients, ICU types, and common consequents

### ISSUE THREAD

**Issue by erikrtn** [erikrtn/bmed8813_icu#50]
Let's focus on the following ICU type, race, and gender for our experiments:

Person | parameters
--- | ---
Dolu | PICU, white male
Aaron | CICU, black male
Erik | NICU, black female

Run three experiments with the following input consequents (see table below):

1. High glucose
2. High glucose & low FIO2
3. High glucose & low FIO2 & high BUN

Lab name | flag value | clinical reason
--- | --- | ---
`POC GLUCOSE` | high | hyperglycemia
`PATIENT FI02` | low | respiratory distress
`BUN` | high | renal dysfunction

We will always choose `7` for the day.

In summary, each of us should have results for 3 experiments, for a total of 12.

Please upload the following results to the [`results folder in Google Drive`](https://drive.google.com/drive/folders/0Bz3v8f9fkd35dWVOR293ZnVSNms):

1. Plot of antecedents generated by GUI.
2. ...
